modules/ossm-vs-istio.adoc

```diff
@@ -12,25 +12,29 @@ The current release of {ProductName} differs from the current upstream Istio com
 [id="ossm-multi-tenant-install_{context}"]
 == Multi-tenant installations
 
-[NOTE]
-====
-Single-tenant control plane installations are known to cause issues with {product-title} restarts and upgrades. Multi-tenant control plane installations are the default configuration starting with {ProductName} {ProductVersion}.
-====
+{ProductName} allows you to configure multi-tenant control plane installations, specify the projects that can access the {ProductShortName}, and isolate the {ProductShortName} from other control plane instances.
 
-{ProductName} allows you to configure multi-tenant control plane installations, specify the namespaces that can access its {ProductShortName}, and isolate the {ProductShortName} from other control plane instances.
+[id="ossm-mt-vs-clusterwide_{context}"]
+=== Differences between multi-tenant and cluster-wide installations
+
+The main difference between a multi-tenant installation and a cluster-wide installation is the scope of privileges used by the control plane deployments, for example, Galley and Pilot. The components no longer use cluster-scoped Role Based Access Control (RBAC) `ClusterRoleBinding`, but rely on project-scoped RBAC `RoleBinding`.
+
+Every project in the `members` list will have a `RoleBinding` for each service account associated with a control plane deployment and each control plane deployment will only watch those member projects. Each member project has a `maistra.io/member-of` label added to it, where the `member-of` value is the project containing the control plane installation.
 
 [id="ossm-automatic-injection_{context}"]
 == Automatic injection
-The upstream Istio community installation automatically injects the sidecar to namespaces you have labeled.
+The upstream Istio community installation automatically injects the sidecar to projects you have labeled.
```

Comment:
semantics, but upstream injects the sidecar to pods in the projects you have labeled.

Reply:
Same with below. Sidecar is a pod level concept, not a project level concept.